repose-aggregator/tests/functional-tests/src/integrationTest/groovy/features/core/powerfilter/RequestSizeTest.groovy

```diff
@@ -43,7 +43,7 @@ class RequestSizeTest extends ReposeValveTest {
         deproxy.addEndpoint(properties.targetPort)
     }
 
-    @Unroll("request with header size of #headerSize should respond with 413")
+    @Unroll("request with header size of #headerSize should respond with 431")
```

Comment:
This is the last behavior change. [RFC 2616](https://tools.ietf.org/html/rfc2616) was also updated by [RFC 6585](https://tools.ietf.org/html/rfc6585) to add additional HTTP Status Codes. The one in particular that is now in use by Jetty v9.3.x+ is `431` which indicates that the server is unwilling to process the request because its header fields are too large.